components/brave_shields/browser/ad_block_regional_service_manager.cc

```diff
@@ -339,7 +339,7 @@ AdBlockRegionalServiceManager::GetRegionalLists() {
         !cookie_list_touched) {
       enabled = true;
     } else if (regional_filter_dict) {
-      regional_filter_dict->GetBoolean("enabled", &enabled);
+      enabled = regional_filter_dict->FindBoolPath("enabled").value_or(false);
```

Comment:
`FindBoolKey`

Reply:
Fixed in 2713c0a45d9d84f7cb4a310c982e48a7064bd900